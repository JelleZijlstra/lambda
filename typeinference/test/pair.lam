let x = (1, \x. 2) in
let y = print (fst x) in
let y = print ((snd x) 42) in
x
