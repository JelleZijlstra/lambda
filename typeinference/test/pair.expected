%%stdout%%
1
2
Result: module
	let __result__ = (1, \x. (2 : int))
end
%%stderr%%
